app/controllers/users_controller.rb

```diff
@@ -1,2 +1,3 @@
 class UsersController < ApplicationController
+  def show; end
```

Comment:
- [ ] Please add the UserControler index action `def index; end` for your app to work properly